src/mds/OpenFileTable.cc

```diff
@@ -636,6 +664,10 @@ void OpenFileTable::commit(MDSContext *c, uint64_t log_seq, int op_prio)
   } else {
     submit_ops_func();
   }
+  logger->inc(l_oft_omap_total_objs, omap_num_objs);
```

Comment:
No, this should be a `set`. We want it to be the "current number of objects" backing the OFT.